tests/factories.py

```diff
@@ -391,6 +391,15 @@ def create_request_at_status(
         return request
 
     if status == RequestStatus.RELEASED:
+        for relpath in request.output_files():
+            bll._dal.release_file(
```

Comment:
I get that we can't actually release files, so we can't call the BLL methods here.

I will try figure out a way in my current refactor.